types/react-dom/experimental.d.ts

```diff
@@ -34,3 +34,12 @@ export {};
 
 declare module "." {
 }
+
+declare module "react" {
+    interface ViewTransitionInstance {
```

Comment:
Name not final: https://github.com/facebook/react/pull/32038#discussion_r1910116833

Reply:
final. Flow type also uses `ViewTransitionInstance`